README.md

```diff
@@ -10,14 +10,27 @@
 
 Swift toolkit that lets you communicate efficiently with many of the [Auth0 API](https://auth0.com/docs/api/info) functions and enables you to seamlessly integrate the Auth0 login.
 
+## Important Notice
+Behaviour changes in iOS 13 related to Web Authentication require that developers using Xcode 11 with this library **must** compile using Swift 5.x. This *should* be the default setting applied when updating, unless it has been manually set. However, we recommend checking that this value is set correctly.
```

Comment:
> Behaviour changes in iOS 13

Feels like we could provide a link here as evidence of the behaviour changes you mention, to give the reader more detail if they're unaware as to what changes you're talking about.

Reply:
Done in aacf3ce99941bb8d7e85e0fd3361b367b77e0e77.